cmd/fs-v1-multipart.go

```diff
@@ -324,11 +330,11 @@ func (fs *FSObjects) PutObjectPart(ctx context.Context, bucket, object, uploadID
 
 	// Should return IncompleteBody{} error when reader has fewer
 	// bytes than specified in request header.
+	// Avoid the check if compression is enabled.
```

Comment:
Do not mention about compression at object layer.

Reply:
I need the have the object locked before doing this ? how do i do this in handlers ?